### ISSUE TITLE
[Feature] Allow running bits based on time

### ISSUE THREAD

**Issue by zcrt** [minvws/nl-kat-octopoes#24]
### Describe the bug, issue or concern

Currently, a Bit runs when an object it consumes has changed. However, some bits should run even though the object has not changed because other things related to the object have changed. A good example is checking the expiry date of a certificate (https://github.com/minvws/nl-kat-coordination/issues/86). The certificate itself has not changed, but compared to yesterday it is closer to expiry nonetheless.

### To Reproduce
Try scheduling a Bit based on time (e.g. daily).

### Expected behavior
The option for a bit to run regularly (which can be different per Bit (daily/weekly/...)).

### Governance
- [x] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

**Comment by underdarknl** [comment]
Agreed. We should be able to trigger certain classes of bits to be re-evaluated based on an external signal (like the date changing). Lets do a brainstorm of what signals could be useful and how we could tell Octopoes which signal corresponds to which BIT when it ingests them.
